hs_core/templates/pages/genericresource.html

```diff
@@ -425,16 +425,16 @@
             </div>
         {% endif %}
         <div id="permission-panel">
-            {% if user == cm.user %}
+            {% if is_owner_user %}
                 <p>You are the owner of this resource.</p>
             {% elif is_edit_user %}
                 <p>You have been given specific permission to edit this resource.</p>
-            {% elif in_edit_group %}
-                Your research group(s) have been given specific permission to edit this resource.</p>
+{#            {% elif in_edit_group %}#}
+{#                Your research group(s) have been given specific permission to edit this resource.</p>#}
             {% elif is_view_user %}
                 <p>You have been given specific permission to view this resource.</p>
-            {% elif in_view_group %}
-                <p>Your research group(s) have been given specific permission to view this resource.</p>
+{#            {% elif in_view_group %}#}
+{#                <p>Your research group(s) have been given specific permission to view this resource.</p>#}
```

Comment:
Why are these commented out now? If we are not distinguishing between having these permissions as a user or a member of a group, then just remove these lines rather than leaving the deadcode commented out.

Otherwise +1 on the new changes.